CalibPPS/ESProducers/plugins/PrintTotemDAQMapping.cc

```diff
@@ -30,33 +30,34 @@ class PrintTotemDAQMapping : public edm::one::EDAnalyzer<> {
 private:
   /// label of the CTPPS sub-system
   std::string subSystemName;
-  edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken_;
-  edm::ESGetToken<TotemAnalysisMask, TotemReadoutRcd> maskToken_;
+  edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken;
+  edm::ESGetToken<TotemAnalysisMask, TotemAnalysisMaskRcd> maskToken;
   void analyze(const edm::Event &e, const edm::EventSetup &es) override;
 };
 
 PrintTotemDAQMapping::PrintTotemDAQMapping(const edm::ParameterSet &ps)
     : subSystemName(ps.getUntrackedParameter<std::string>("subSystem")),
-      mappingToken_(esConsumes(edm::ESInputTag("", subSystemName))),
-      maskToken_(esConsumes(edm::ESInputTag("", subSystemName))) {}
+      mappingToken(esConsumes(edm::ESInputTag("", subSystemName))),
+      maskToken(esConsumes(edm::ESInputTag("", subSystemName))) {}
 
 //----------------------------------------------------------------------------------------------------
 
 void PrintTotemDAQMapping::analyze(const edm::Event &, edm::EventSetup const &es) {
   // get mapping
-  auto const &mapping = es.getData(mappingToken_);
+  if (auto mappingHandle = es.getHandle(mappingToken)) {
+    auto const &mapping = *mappingHandle;
+    edm::LogPrint("PrintTotemDAQMapping") << mapping;
+  } else {
+    edm::LogError("PrintTotemDAQMapping mapping") << "No mapping found";
```

Comment:
the body of the `edm::LogError` should have the name of the module, i.e. `PrintTotemDAQMapping`